Bio/ExPASy/Cellosaurus.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999 by Jeffrey Chang.  All rights reserved.
+# Copyright 2009 by Michiel de Hoon.  All rights reserved.
```

Comment:
This needs changing - your name is missing for a start. Was this heavily based on code by Michiel or Jeff?